tests/integration/socialAuth/index.test.js

```diff
@@ -0,0 +1,13 @@
+import chai from "chai";
+import { dummyUser } from "../../helpers/socialAuthHelpers";
+import createUserFromSocial from "../../../controllers/auth/socials/createUserFromSocial";
+
+const { expect } = chai;
+
+it("check if social login callback is workings", async () => {
```

Comment:
Correct that spelling error `... login callback is working` not `... is workings`